sky/jobs/state.py

```diff
@@ -342,6 +342,8 @@ class ManagedJobScheduleState(enum.Enum):
     briefly observe inconsistent states, like a job that just finished but
     hasn't yet transitioned to DONE.
     """
+    # This job may have been created before the column was present.
```

Comment:
```suggestion
    # This job may have been created before scheduler was introduced in #4458.
    # This state is not used by scheduler but just for backward compatibility.
    # TODO(cooperc): remove this in v0.11.0
```
Also, should we update the docstr above as well?